src/components/safe-apps/AppFrame/TransactionQueueBar/index.tsx

```diff
@@ -22,7 +22,7 @@ const TransactionQueueBar = ({ expanded, visible, setExpanded, onDismiss }: Prop
   const { page = { results: [] } } = useTxQueue()
   const queuedTxCount = getQueuedTransactionCount(page)
 
-  if (!visible) {
+  if (!visible || queuedTxCount === '0') {
```

Comment:
I would suggest we compare against the `results?.length` instead. We could change the return of `getQueuedTransactionCount` (to i.e. "No transactions found") and this would break.

Reply:
Thanks ser, adjusted it now